src/core/worker.js

```diff
@@ -436,6 +437,55 @@ var WorkerMessageHandler = PDFJS.WorkerMessageHandler = {
       });
     });
 
+    handler.on('GetRawObject', function wphGetRawObject(data) {
+      function clone(obj) {
+        if (!obj || isBool(obj) || isNum(obj) || isString(obj)) {
+          return obj;
+        } else if (isName(obj)) {
+          return obj;
+        } else if (isDict(obj)) {
+          var clonedMap = {};
+          var map = obj.map;
+          for (var k in map) {
+            clonedMap[k] = clone(map[k]);
+          }
+          return { map: clonedMap };
+        } else if (isArray(obj)) {
+          var clonedArray = [];
+          for (var i = 0, ii = obj.length; i < ii; i++) {
+            clonedArray.push(clone(obj[i]));
+          }
+          return clonedArray;
+        } else if (isStream(obj)) {
+          if (obj.str) {
+            return clone(obj.str);
+          } else if (obj.stream) {
+            return clone(obj.stream);
+          } else if (obj.streams) {
+            return clone(obj.streams);
+          }
+          return { dict: clone(obj.dict), start: obj.start, end: obj.end };
+        } else if (isRef(obj)) {
+          return obj;
```

Comment:
No, it should be `return { num: obj.num, gen: obj.gen }` since it is a reference object.